src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -7,23 +7,24 @@ namespace FirstSteps.RPG.Heroes
 {
     public abstract class Hero
     {
-        private List<Item> backPack = new List<Item>();
-        private int _coinsBag;
+        private Equipment _equipment;
         private string _name;
         private Races _race;
         protected int _strength;
         protected int _intelligence;
         protected int _agility;
         protected int _health;
-        protected int _damage;
-        private int maxBackpack = 5;
-        private int _maxWeight = 40;
-        public int Coins { get { return _coinsBag; } }
+        protected int _damage;   
         public Hero(string name, Races race)
         {
+            _equipment = new Equipment(); // i found a similar example i thnk on stackoverflow. can i do it some other way? with variable. Can u show me other example/ solution? 
             _name = name;
             _race = race;
         }
+        public Equipment GetEquipment()
```

Comment:
It's not the best solution. Look at this example of encapsulation (you can read more about it):
```csharp
public class Person
{
    private string _name; 
    private Address _address; 

    public Person(string name, Address address)
    {
        _name = name;
        _address = address;
    }

    public void ChangeCity(string city)
    {
        return _address.City = city;
    }
}

public class Address
{
    private string Street { get; set; }; 
    private string City { get; set; };
    private string Country { get; set; };

    public Address(string street, string city, string country)
    {
        Street = street;
        City = city;
        Country = country;
    }
}
```

The person has an address and we need to add the ability to change the city. So instead of making `public Address GetAddress()` we can create a `ChangeCity(string city)` and just use private `_address` in the method implementation. And you can use the same approach for equipment, for example, `public int Coins { get { return _equipment.Coins; } }`